tests/ndc/cluster_config.go

```diff
@@ -0,0 +1,67 @@
+package ndc
+
+import (
+	"go.temporal.io/server/common/cluster"
+	"go.temporal.io/server/tests/testcore"
+)
+
+func clustersConfig() []*testcore.TestClusterConfig {
+	// TODO (alex): I don't know how many of these settings are really needed.
+
+	clusterInformation := map[string]cluster.ClusterInformation{
+		"cluster-a": {
+			Enabled:                true,
+			InitialFailoverVersion: 1,
+			RPCAddress:             "nowhere1:7134",
```

Comment:
I think these are either not needed now, or can be dropped with a tiny bit of work. but no need to do it now.

Reply:
Simple removal doesn't work, I checked.